app/jobs/prepare_move_notifications_job.rb

```diff
@@ -31,12 +31,11 @@ def perform(topic_id:, action_name:)
 private
 
   def build_notification_id(subscription, type_id, topic, action_name)
-    { notification_id:
-          subscription.notifications.create!(
-            notification_type_id: type_id,
-            topic: topic,
-            event_type: event_type(action_name),
-          ).id }
+    subscription.notifications.create!(
+      notification_type_id: type_id,
+      topic: topic,
+      event_type: event_type(action_name),
+    ).id
```

Comment:
should this signature change: previously it was returning a hash, now it is return an id ?

Reply:
Yep, that's intentional. I removed the single named argument on the job and just changed it to a regular parameter to fix the deprecation warning. It didn't seem necessary to name the argument when there's only one, so this removes the hash wrapper and just returns the id.